theme/less/_content.less

```diff
@@ -1,4 +1,62 @@
-main article img
+article
+{
+  font-size: 1rem;
+  color: #444444;
+  font-weight: 400;
+  line-height: 1.5;
+}
+
+article img
 {
   max-width: 100%;
 }
+
+article p:first-of-type img
+{
+  margin-top: 1em;
+}
+
+article h1
+{
+  font-size: 2.19em;
+  line-height: 1.4em;
+}
+
```

Comment:
Meeting-note:
- Let's specify margin for `article p` where margin-bottom is smaller, for the cases when we have following ul.
- And also specify margins for article headings.